### PR TITLE
December 2022 k3s - Add 1.25.5 to v2.7

### PR DESCRIPTION
**Pull request by nikolaishields**

### CODE REVIEW

**Review by brandond — DISMISSED** [review]

File: channels.yaml [hunk 1]
```diff
@@ -309,6 +309,13 @@ releases:
     serverArgs: *serverArgs-v3
     agentArgs: *agentArgs-v2
     featureVersions: *featureVersions-v1
-
   #  v1.25.1+k3s1 was never released
   #  v1.25.2+k3s1 is not released through KDM yet
+  - version: v1.25.5-rc3+k3s1
+    minChannelServerVersion: v2.6.7-alpha1
```
Comment (rancher-max) [inline channels.yaml]: This should be `v2.7.2-alpha1`

File: channels.yaml [hunk 2]
```diff
@@ -309,6 +309,13 @@ releases:
     serverArgs: *serverArgs-v3
     agentArgs: *agentArgs-v2
     featureVersions: *featureVersions-v1
-
   #  v1.25.1+k3s1 was never released
   #  v1.25.2+k3s1 is not released through KDM yet
+  - version: v1.25.5-rc3+k3s1
```
Comment (rancher-max) [inline channels.yaml]: Let's have this be our currently released version: `v1.25.4+k3s1`

**Review by rancher-max — CHANGES_REQUESTED** [review]

File: channels.yaml (hunk 1 above)
Comment (nikolaishields) [reply channels.yaml]: Thanks for the heads up, this is now complete 👍

**Review by rancher-max — APPROVED** [review]
LGTM. Thank you!

**Review by dereknola — APPROVED** [review]

**Pull request merged**